src/util/error.h

```diff
@@ -32,9 +32,9 @@ enum class TransactionError {
     MAX_FEE_EXCEEDED,
 };
 
-std::string TransactionErrorString(const TransactionError error);
+bilingual_str TransactionErrorString(const TransactionError error);
```

Comment:
in commit 2b9114768bb842a82ef82f317d7f0408267e7dc5 : 

I can no longer compile the fuzz tests:

```
test/fuzz/kitchen_sink.cpp:24:11: error: calling 'TransactionErrorString' with incomplete return type 'bilingual_str'

    (void)TransactionErrorString(transaction_error);
```

It might be possible to fix this by including `util/translation.h` in the fuzz test that fails to compile

Reply:
confirmed here as well
```
test/fuzz/kitchen_sink.cpp:24:11: error: calling 'TransactionErrorString' with incomplete return type 'bilingual_str'
    (void)TransactionErrorString(transaction_error);
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
./util/error.h:35:15: note: 'TransactionErrorString' declared here
bilingual_str TransactionErrorString(const TransactionError error);
              ^
./util/error.h:20:8: note: forward declaration of 'bilingual_str'
struct bilingual_str;
       ^
1 error generated.
```
and verified that your suggestion allows the test to build and run correctly.